Brokerages/Paper/PaperBrokerageFactory.cs

```diff
@@ -53,6 +47,24 @@ public override IBrokerageModel BrokerageModel
         /// <returns>A new brokerage instance</returns>
         public override IBrokerage CreateBrokerage(LiveNodePacket job, IAlgorithm algorithm)
         {
+            // special handling for GDAX paper trading
+            if (job.DataQueueHandler.EndsWith("GDAXDataQueueHandler"))
```

Comment:
@StefanoRaggi  - Have we given any additional thought towards a better way of supporting brokerage data queue handlers running with the `PaperBrokerage`? The solution here won't scale well as we increase the number of brokerages.